pram

```diff
@@ -232,6 +261,13 @@ main() {
 			;;
 		*)
 			# a local file maybe?
+			if [[ ${partof} == def && -z ${link_to} ]]; then
+				# only add partof for local files if explicitly asked for
+				partof=
+			fi
+			[[ ! ${partof} || -n ${link_to} ]] ||
+				die "specifying --link-to is mandatory with local files when using --part-of"
```

Comment:
Combine both into `if [[ -z ${link_to} ]]`.

Reply:
```bash
if [[ -z ${link_to} ]]; then
	# only add partof for local files if explicitly asked for
	if [[ ${partof} == def ]]; then
		partof=
	fi
	[[ ! ${partof} ]] ||
		die "specifying --link-to is mandatory with local files when using --part-of"
fi
```

Trying to figure out the best way to do this, keeping with the general style of the script. I think this fits best, though the way I'd have written it is with the die in an `elif` (or `[[ ${partof} ]] && die`).

If you've got a different way you'd prefer, I can commit that instead; asking here to avoid a little extra back-and-forth...

Reply:
Alright, done!